src/screens/DetailedViewScreen.tsx

```diff
@@ -46,25 +59,66 @@ const DetailedViewScreen: React.FC<PlaceProps> = ({ setPage, placeID }: PlacePro
 						height: "40%",
 					}}
 				/>
-				<View style={styles.content}>
-					<Text ellipsizeMode="tail" numberOfLines={2} style={styles.title}>
-						{place.name}
-					</Text>
-					<BodyText ellipsizeMode="tail" numberOfLines={2}>
-						{place.formatted_address}
-					</BodyText>
-					<BodyText accessibilityLabel={distanceInMi ? `${distanceInMi} miles away` : ""}>
-						{distanceInMi ? `${distanceInMi} mi away` : ""}
-					</BodyText>
-					<BodyText>
-						{Math.round(
-							getAverageA11yRating({
-								accessibilityData: placeDetails.placeDetails.accessibilityData,
-							}) * 2
-						) / 2}
-					</BodyText>
+				<View
+					style={{
+						width: "90%",
+						alignSelf: "center",
+					}}
+				>
+					<ScrollView style={{ height: "40%" }}>
+						<Text
+							ellipsizeMode="tail"
+							numberOfLines={2}
+							style={{
+								fontSize: 35,
+								fontWeight: "bold",
+							}}
+						>
+							{place.name}
+						</Text>
+						<BodyText ellipsizeMode="tail" numberOfLines={2}>
+							{place.formatted_address}
+						</BodyText>
+						<BodyText
+							accessibilityLabel={distanceInMi ? `${distanceInMi} miles away` : ""}
+						>
+							{distanceInMi ? `${distanceInMi} mi away` : ""}
+						</BodyText>
+						{[
+							"Navigability",
+							"Sensory Aids",
+							"Staff Helpfulness",
+							"Guide Dog Friendliness",
+						].map((value, index) => {
```

Comment:
This would be significantly improved by combining the labels and a11yDataList from above into one object, basically a map, and then loop through `Object.entries(<thatMap>)`